install-dev/models/install.php

```diff
@@ -114,21 +116,14 @@ public function generateSettingsFile(
                     'database_engine' =>  $database_engine,
                     'cookie_key' => $cookie_key,
                     'cookie_iv' =>  $cookie_iv,
+                    'new_cookie_key' => $key->saveToAsciiSafeString(),
```

Comment:
Ohh, but wait, the previous one is actually used as a salt. How about renaming the previous one and call this the cookie key instead?